src/core/memory.cc

```diff
@@ -1324,6 +1491,14 @@ void disable_large_allocation_warning() {
 }
 
 void configure(std::vector<resource::memory> m, bool mbind, bool hugepages) {
```

Comment:
With this change I think we should remove the configure calls from our non-seastar SQL threads

Reply:
yeah, i'm working on trying to get SQL to compile